src/icon-manifest/languages.ts

```diff
@@ -3,6 +3,7 @@ import { ILanguageCollection } from '../models';
 export const languages: ILanguageCollection = {
   actionscript: { ids: ['nextgenas'], defaultExtension: 'as' },
   ansible: { ids: ['ansible'], defaultExtension: 'ansible' },
+  anyscript: { ids: ['anyscript'], defaultExtension: 'any'},
```

Comment:
Move it after `ansible` to keep alphabetic ordering. Don't let `ng` mislead you they are there because they are `angular` related.